### PR TITLE
Extend ground workflow

### PR DESCRIPTION
**Pull request by keskitalo**

### CODE REVIEW

File: src/toast/tests/ops_sss.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        # Create fake observing of a small patch
+        data = create_ground_data(self.comm)
+
+        # Simple detector pointing
+        detpointing_azel = ops.PointingDetectorSimple(
+            boresight=obs_names.boresight_azel, quats="quats_azel"
+        )
+
+        # Flag
```
Comment (tskisner) [inline src/toast/tests/ops_sss.py:46]: Not sure this comment makes sense...

File: src/toast/ops/cadence_map.py [hunk 2]
```diff
@@ -84,19 +89,26 @@ def __init__(self, **kwargs):
...
+                raise RuntimeError(msg)
 
         comm = data.comm.comm_world
         rank = data.comm.world_rank
 
-        # Get the total number of pixels from the pixel distribution
+        # We need the pixel distribution to know total number of pixels
+
+        if self.pixel_dist not in data:
+            pix_dist = BuildPixelDistribution(
```
Comment (tskisner) [inline src/toast/ops/cadence_map.py:103]: Thanks, much cleaner!

**Review by tskisner — APPROVED** [review]
Looks good, just a small comment about stale comments on an internal function.

**Comment by keskitalo** [comment]
Fixed stale comments. Merging.

**Pull request merged**